### PR TITLE
Fix documentation and setting tags

### PR DESCRIPTION
**Pull request by karthiknadig**
Closes https://github.com/microsoft/vscode-mypy/issues/144

### CODE REVIEW

File: .vscode/settings.json
Comment (anthonykim1) [inline .vscode/settings.json:1]: I think there are two instances of "editor.formatOnSave": true, for some reason

Comment (karthiknadig) [reply .vscode/settings.json:1]: Once is language specific setting and the other is a general setting.

Comment (anthonykim1) [reply .vscode/settings.json:1]: great, thanks for explaining!

**Review by anthonykim1 — APPROVED** [review]

File: .vscode/settings.json
Comment (anthonykim1) [reply .vscode/settings.json:1]: when there is language specific setting, does the general setting get overriden?

Comment (karthiknadig) [reply .vscode/settings.json:1]: Yes, you can override.

File: README.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ If you want to disable mypy, you can [disable this extension](https://code.visua
...
+| mypy-type-checker.path              | `[]`                                          | Setting to provide custom `mypy` executable. This will slow down linting, since we will have to run `mypy` executable every time or file save or open. Example 1: `["~/global_env/mypy"]` Example 2: `["conda", "run", "-n", "lint_env", "python", "-m", "mypy"]`                                                                                |
+| mypy-type-checker.interpreter       | `[]`                                          | Path to a python interpreter to use to run the linter server. When set to `[]` interpreter for the workspace is obtained from `ms-python.python` extension. If set to some value, that value takes precedence, and Python extension is not queried for the interpreter.                                                                          |
```
Comment (luabud) [inline README.md]:
```suggestion
| mypy-type-checker.interpreter       | `[]`                                          | Path to a python interpreter to use to run the linter server. When set to `[]`, the interpreter for the workspace is obtained from `ms-python.python` extension. If set to some path, that path takes precedence, and the Python extension is not queried for the interpreter.                                                                         |
```

**Review by anthonykim1 — APPROVED** [review]

**Pull request merged**